cmd/deploy.go

```diff
@@ -41,7 +41,7 @@ or -n flag, and if so this will overwrite the value in the faas.yaml file.
 
 func runDeploy(cmd *cobra.Command, _ []string) (err error) {
 	config := newDeployConfig()
-	function, err := functionWithOverrides(config.Path, config.Namespace, "")
+	function, err := functionWithOverrides(config.Path, overrideNamespace(config.Namespace))
```

Comment:
Assume this is unrelated, but was in fact a bug?

Reply:
No it wasn't, empty string is ignored, so it's OK. I choose to change the function to be variadic, because I introduced yet another override: `Builder`.

Reply:
I didn't wanted to end up with:
```go
function, err := functionWithOverrides(config.Path, config.Namespace, "", "")
```
Plus it's not to difficult to mistake parameter order in the previous version.